Libs/vtkSegmentationCore/vtkSegmentation.h

```diff
@@ -304,7 +308,63 @@ class vtkSegmentationCore_EXPORT vtkSegmentation : public vtkObject
   /// Invalidate (remove) non-master representations in all the segments if this segmentation node
   void InvalidateNonMasterRepresentations();
 
-// Conversion related methods
+// Merged labelmap utility functions
+
+  /// Returns true if the binary labelmap representation is merged
+  bool IsBinaryLabelmapMerged(std::string segmentID);
+
+  void GetMergedSegmentIDsForRepresentation(vtkSegment* segment, std::string representationName,
+    std::vector<std::string>& sharedSegmentIds, bool includeMainSegmentId);
+  void GetMergedSegmentIDsForRepresentation(std::string segmentId, std::string representationName,
+    std::vector<std::string>& sharedSegmentIds, bool includeMainSegmentId);
+  void GetMergedBinaryLabelmapSegmentIDs(vtkSegment* segment, std::vector<std::string> &sharedSegmentIds,
+    bool includeMainSegmentId);
+  void GetMergedBinaryLabelmapSegmentIDs(std::string segmentId, std::vector<std::string> &sharedSegmentIds,
+    bool includeMainSegmentId);
+  int GetUniqueValueForMergedLabelmap(std::string segmentId);
+  int GetUniqueValueForMergedLabelmap(vtkOrientedImageData* labelmap);
+  void MergeSegmentLabelmaps(std::vector<std::string> mergeSegmentIds);
+#ifndef __VTK_WRAP__
+  //BTX
+  bool GenerateMergedLabelmap(vtkOrientedImageData* mergedImageData, int extentComputationMode, vtkOrientedImageData* mergedLabelmapGeometry = nullptr,
+    const std::vector<std::string>& segmentIDs = std::vector<std::string>());
+  ///ETX
```

Comment:
BTX/ETX is deprecated. I don't remember what's instead, because in every case I could safely remove it. Please try just removing it.
Other than that BTX has only 2 slashes while ETX has three, in case it needs to stay (remove third one from ETX)